flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroDeserializationSchema.java

```diff
@@ -53,22 +55,25 @@ public class AvroDeserializationSchema<T> implements DeserializationSchema<T> {
      * schema.
      *
      * @param schema schema of produced records
+     * @param encoding Avro serialization approach to use for decoding
      * @return deserialized record in form of {@link GenericRecord}
      */
-    public static AvroDeserializationSchema<GenericRecord> forGeneric(Schema schema) {
-        return new AvroDeserializationSchema<>(GenericRecord.class, schema);
+    public static AvroDeserializationSchema<GenericRecord> forGeneric(
```

Comment:
This API is public and all changes need to be backwards compatible.

Reply:
Thanks very much for catching that - I've added a wrapper for these methods that maintains the previous API signature. I'd done that for constructors but missed these, apologies.